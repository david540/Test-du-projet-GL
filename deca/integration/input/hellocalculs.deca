{
        println((3 * 2 + 4 - 5)/3 % 2);
        println((3.5 * 2 + 4 - 5)/3);
}
