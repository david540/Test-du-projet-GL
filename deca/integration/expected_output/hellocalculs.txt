1
2.00000e+00
